### PR TITLE
Other styles

### PR DESCRIPTION
**Pull request by rondoor124**
Change styles of `Components` and `Containers` folder files, except for subdirectories.

Demo Link: https://deploy-preview-524--bldrs-share.netlify.app/

`PTAL`

### CODE REVIEW

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
Ok, reabase looks to have worked.  Now for review..

The avatar icon break is back:
image


Seeing a new scrollbar in Share dialog:
image

also seeing new warning in the console
image

Please fix these up and I'll do a code review.  Thanks!

**Comment by pablo-mayrgundter** [comment]
Ok, icons look fixed!  Also not seeing the console error anymore.

New ones:
Scrollbars in mobile:
image

lost some spacing in Open dialog:
image

**Comment by pablo-mayrgundter** [comment]
Ok, maybe have a repro for the console warning:

1) Open comments,
2) Open properties,
3) Close properties -> console warning

Complaining about anchorEl

image

**Comment by rondoor124** [comment]
Hi @pablo-mayrgundter.
I addressed all issues you mentioned, please review.
the e2e test works well locally, but get error occasionally on live. seems due to cypress disadvantage.
please let me know the next steps. thank you.

File: src/utils/itemProperties.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, {useState} from 'react'
 import debug from './debug'
-import Typography from '@mui/material/Typography'
+import {Typography} from '@mui/material'
```
Comment (pablo-mayrgundter) [inline src/utils/itemProperties.jsx:3]: Revert

File: src/Components/Toggle.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, {useContext} from 'react'
-import Switch from '@mui/material/Switch'
+import {Switch} from '@mui/material'
```
Comment (pablo-mayrgundter) [inline src/Components/Toggle.jsx:2]: Revert

File: src/Components/SnackbarMessage.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, {useContext} from 'react'
+import {Snackbar} from '@mui/material'
 import MuiAlert from '@mui/material/Alert'
-import Snackbar from '@mui/material/Snackbar'
```
Comment (pablo-mayrgundter) [inline src/Components/SnackbarMessage.jsx:3]: Revert

File: src/Components/SideDrawer/SideDrawerPanels.jsx [hunk 4]
```diff
@@ -1,7 +1,5 @@
 import React from 'react'
-import Box from '@mui/material/Box'
-import Typography from '@mui/material/Typography'
-import {makeStyles, useTheme} from '@mui/styles'
+import {Box, Typography} from '@mui/material'
```
Comment (pablo-mayrgundter) [inline src/Components/SideDrawer/SideDrawerPanels.jsx:2]: Revert

**Review by pablo-mayrgundter — APPROVED** [review]
Sorry, just noticed you've moved from default to named imports for all the mui components.  Please move back to defaults as per this note:

Ihttps://mui.com/material-ui/guides/minimizing-bundle-size/

We've tested this out and it does save a few hundred KB on the bundle.

I test locally for any named imports with:

```
> grep -r \'@mui/material\' src
src/Components/Dialog.jsx:import {Typography} from '@mui/material'
src/Components/AboutControl.jsx:import {Box} from '@mui/material'
```

And then edit them back to default.

File: src/utils/itemProperties.jsx (hunk 1 above)
Comment (rondoor124) [reply src/utils/itemProperties.jsx]: I have done.

File: src/Components/Toggle.jsx (hunk 2 above)
Comment (rondoor124) [reply src/Components/Toggle.jsx]: I have done.

File: src/Components/SnackbarMessage.jsx (hunk 3 above)
Comment (rondoor124) [reply src/Components/SnackbarMessage.jsx:3]: I have done.

File: src/Components/SideDrawer/SideDrawerPanels.jsx (hunk 4 above)
Comment (rondoor124) [reply src/Components/SideDrawer/SideDrawerPanels.jsx]: I have done.

**Comment by rondoor124** [comment]
@pablo-mayrgundter, I reverted all import method to the original status.
Please check the update, and give me feedback.
Thank you. :)

**Pull request merged**